Automattic/wp-calypso
# Visually Improve the Push Notification Authentication Step

#### Changes proposed in this Pull Request

When two-factor authentication is enabled:

* Visually emphasize the fact that the flow requires opening a notification from the mobile app.
* Make it clearer that the user can choose to continue with an alternative action (using their own authenticator app or by getting a code via text).

Meta:

* Add support for `<Button block />` that removes the necessity for superfluous module-specific class names.

#### Testing instructions

* Make sure two-factor authentication is enabled on your account.
* Log out if you’re signed in.
* Log in.
* When the push notification step appears:
    * Make sure it works exactly like on [the video](https://cloudup.com/coX_t6HnAoo) and looks like on the screenshot below.
    * The push notification illustration animates in.

Repeat in all the supported browsers.

Fixes #33550.

[<img width="1241" alt="Screenshot" src="https://user-images.githubusercontent.com/2060580/59400522-85015a00-8d4c-11e9-810f-fdd5fc541da4.png">](https://cloudup.com/coX_t6HnAoo)
